sql/nycha_communitycenters.sql

```diff
@@ -0,0 +1,40 @@
+--select w.status::text, count(*) 
+--from (select geo::json->'status' as status from bpl_libraries) w
+--group by w.status::text;
+
+ALTER TABLE nycha_communitycenters
+	ADD hash text, 
+	ADD	facname text,
+	ADD	factype text,
+	ADD	facsubgrp text,
+	ADD	facgroup text,
+	ADD	facdomain text, 
+	ADD	servarea text,
+	ADD	opname text,
+	ADD	opabbrev text,
+	ADD	optype text,
+	ADD	overagency text,
+	ADD	overabbrev text,
+	ADD	overlevel text,
+	ADD	capacity text,
+	ADD	captype text,
+	ADD	proptype text;
+
+update nycha_communitycenters as t
+SET hash =  md5(CAST((t.*)AS text)), 
```

Comment:
Since there are several entries has coordinates in the raw data didn't be recognized my geosupport, you can use the source data coordinates to fill the blanks in `wkb_geometry` (we don't care about `the_geom`). Here is the code you can add within the update syntax (put it right before `facname = name`):

	wkb_geometry = (CASE
				        WHEN wkb_geometry IS NULL 
							AND longitude != 'None' AND latitude != 'None'
				        THEN ST_SetSRID(ST_Point(longitude::DOUBLE PRECISION, 
									             latitude::DOUBLE PRECISION), 4326)
				        ELSE wkb_geometry
				    END),